src/xrpld/app/tx/applySteps.h

```diff
@@ -29,6 +29,13 @@ class Application;
 class STTx;
 class TxQ;
 
+struct ApplyResult
+{
+    TER ter;
+    bool applied;
```

Comment:
Please add default values: false / tesSuccess looks good

Reply:
I disagree with this - it feels more likely that bugs may be accidentally introduced if there are default values, due to forgetting to initialize them correctly.

Reply:
What the difference between bugs with initialized and bugs with randomly initialized values? You can't distinguish the values.  But with initialized values you can save some code space.

Reply:
My understanding of structs is that it will throw a compiler error if you don't specify them when constructing them. Is that incorrect?

Reply:
Fixed in a897804f389d34495eaec1457455c2e236b7ebcb